woocommerce/google-listings-and-ads
# Improve post-onboarding: Show promotion for Google Ads campaign on the Dashboard page

### Changes proposed in this Pull Request:



Closes #2538.

_Replace this with a good description of your changes & reasoning._


### Screenshots:

<img width="517" alt="image" src="https://github.com/user-attachments/assets/37240a05-6da3-41c9-ad89-de4435d9af9c">

### Detailed test instructions:


1. Complete onboarding by skipping creating a campaign.
2. Go to the dashboard after the MC step. The content should be shown under the Google Ads summary section.
3. Click on the Create Campaign and see if that takes you to the Create Campaign section.
4. Create a campaign, and the section should disappear.

In this task, we have also deprecated the `useFreeAdCredit()`  check the last point in IB. 
Test clicking `Add Paid Campaign`, `Spend $500 to get $500` should only be shown if there is no campaign.


### Additional details:

We have used the common FreeAdCredit component here, which standardises the text and various links associated with that component.


### Changelog entry

> Add - Show promotion for Google Ads campaign on the Dashboard page.
